pkg/util/scrubber/default.go

```diff
@@ -102,7 +102,7 @@ func AddDefaultReplacers(scrubber *Scrubber) {
 	// replace all alphanum + special chars except quotes and semicolon
 	passwordReplacer := Replacer{
 		// see test cases for match patterns
-		Regex: regexp.MustCompile(`(?i)(\"?(?:pass(?:word)?|pswd|pwd)\"?)( = |=|: \"?)([0-9A-Za-z#!$%&()*+,\-./:<=>?@[\\\]^_{|}~]+)`),
```

Comment:
This regex hurts my head a little bit. It certainly covers lots of cases and it's good to have strong test coverage, but I'm not sure if I understand exactly what it does. Could you write a bit of documentation to explain its parts, or at least put a TODO in place to do some soon.

Reply:
fixed in https://github.com/DataDog/datadog-agent/pull/28177/commits/f86540dfc520b8e739c12dfc8b4bdbc8ae6fc489